src/reachability/utils/reachability.jl

```diff
@@ -3,7 +3,7 @@
 # Performs layer-by-layer propagation
 # It is called by all solvers under reachability
 # TODO: also called by ReluVal and FastLin, so move to general utils (or to network.jl)
-function forward_network(solver, nnet::Network, input::AbstractPolytope)
+function forward_network(solver, nnet::Network, input::LazySet)
```

Comment:
What was the reason to convert all of the `::AbstractPolytope` types into `::LazySet`s instead? Are these functions accepting inputs that aren't AbstractPolytopes? Is it a Rectification or AffineMap?

Reply:
My idea was to let the user pass an input with something like `[1.0, 2.0] × [1.0, 2.0]` but it wasn't done in the end, I will change it back